src/main/java/mate/academy/Main.java

```diff
@@ -1,16 +1,85 @@
 package mate.academy;
 
+import java.time.LocalDate;
+import mate.academy.lib.Injector;
+import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
+import mate.academy.model.MovieSession;
 import mate.academy.service.MovieService;
+import mate.academy.service.impl.CinemaHallService;
+import mate.academy.service.impl.MovieSessionService;
```

Comment:
The import statement for `MovieSessionService` is incorrect. It should not be imported from the `impl` package. Ensure that the correct interface or service is being used as per the task requirements.